server/migrations/1_initial_schema.sql

```
@@ -0,0 +1,52 @@
+    max_tokens INTEGER
+);
+
+-- Index for efficient message lookups by conversation
+CREATE INDEX IF NOT EXISTS idx_messages_conversation_id ON messages(conversation_id);
+
+-- Trigger to auto-update updated_at on conversation changes
+CREATE TRIGGER IF NOT EXISTS update_conversations_timestamp
+    AFTER UPDATE ON conversations
+    FOR EACH ROW
+BEGIN
+    UPDATE conversations SET updated_at = CURRENT_TIMESTAMP WHERE id = NEW.id;
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)